core/lib/config/src/configs/proof_data_handler.rs

```diff
@@ -28,6 +33,14 @@ impl TeeConfig {
     pub fn default_first_tee_processed_batch() -> L1BatchNumber {
         L1BatchNumber(0)
     }
+
+    pub fn default_tee_proof_generation_timeout_in_secs() -> u16 {
+        600
+    }
+
+    pub fn tee_proof_generation_timeout(&self) -> Duration {
+        Duration::from_secs(self.tee_proof_generation_timeout_in_secs as u64)
```

Comment:
Nit: You can use `tee_proof_generation_timeout_in_secs.into()` to clearly signal that the conversion is lossless.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/3128/commits/391ce36c36abfe54dc7eea14ad5b9efaef511b06). ✅